test_competitor/src/test_competitor.cpp

```diff
@@ -1267,9 +1462,19 @@ bool TestCompetitor::CompleteKittingTask(ariac_msgs::msg::KittingTask task)
 
   FloorRobotPickandPlaceTray(task.tray_id, task.agv_number);
 
+  bool found;
```

Comment:
Oh I see the check is here. Move this into the PickConveyorPart method

Reply:
I didn't write this inside because we should only enter FloorRobotPickConveyorPart if a part were expected.

Reply:
The bin pick function checks inside of the function so I would like the conveyor function to work similarly.